website/docs/layouts/grid/partials/code/how-to-use.md

```diff
@@ -0,0 +1,284 @@
+    <Doc::Placeholder @height="40px" @text="Item 3" @background="#e4e4e4" />
+    <Doc::Placeholder @height="40px" @text="Item 4" @background="#e4e4e4" />
+  </Hds::Layout::Grid>
+</div>
+```
+
+### isInline
+
+To change the default display from `grid` to `inline-grid`, set `@isInline` to `true`.
+
+```handlebars
+Text before the inline grid.
+<Hds::Layout::Grid  @isInline={{true}} @tag="span" @gap="8">
+  <Doc::Placeholder @height="1.2em" @text="Item 1" @background="#e4e4e4" />
+  <Doc::Placeholder @height="1.2em" @text="Item 2" @background="#e4e4e4" />
+  <Doc::Placeholder @height="1.2em" @text="Item 3" @background="#e4e4e4" />
+  <Doc::Placeholder @height="1.2sem" @text="Item 4" @background="#e4e4e4" />
+</Hds::Layout::Grid>
+Text after the inline grid.
+```
+
+### Grid::Item
+
+The `Grid::Item` component can optionally be used to wrap `Grid` content if more control is needed over the grid layout.
+
+### colSpan & rowSpan
+
+Use the `colSpan` and `rowSpan` options of `Grid::Item` components to set the number of columns or rows an item should occupy.
+
+Note: By default, if a height is set on the `Grid` parent, row heights will stretch proportionally to fill the Grid. To instead make a row conform to the minimum height of its content, you can pass an inline style as shown in the example.
+
+```handlebars
+<div {{style height="400px" border="1px solid"}}>
+  <Hds::Layout::Grid 
+    @columnMinWidth="25%" 
+    @gap="12"
+    {{style height="100%" grid-template-rows="min-content auto auto"}}
+    as |LG|
+  >
+    <LG.Item @colSpan="4">
+      <Doc::Placeholder @height="100%" @text="Item 1" @background="#e4e4e4" {{style padding="1em"}} />
+    </LG.Item>
+
+    <LG.Item @rowSpan="2">
+      <Doc::Placeholder @height="100%" @text="Item 2" @background="#e4e4e4" />
+    </LG.Item>
+
+    <LG.Item @colSpan="3" @rowSpan="2">
+      <Doc::Placeholder @height="100%" @text="Item 3" @background="#e4e4e4" />
+    </LG.Item>
+  </Hds::Layout::Grid>
+</div>
+```
+
+---
+
+## Common layout patterns
+
+Below are examples of common layout patterns that can be achieved using the `Layout::Grid` component in combination with other HDS components.
+
+!!! Warning
+
+**Important**
+
+The examples below are meant to show how one _could_ use the `Layout::Grid` component to implement certain common/standard UI patterns. They're **not** meant to be taken literally as they are and be used in production code. Also, some of these patterns are already implemented directly in HDS components that are ready to use.
+
+!!!
+
+### Card layouts
+
+Note: The following example makes use of nested `Grid` and `Flex` components to achieve its layout. This may be overkill in actual practice but demonstrates the possibilities for achieving layouts with just these layout components alone.
+
+#### Basic layout
+
+**TODO: In HCP, mobile view shows only one column instead of 3, consider adding responsive layout feature to Grid**
```

Comment:
@didoo Do you have any thoughts yet as to whether we should try to include responsive layout features? (As we discussed in a previous sync)

Maybe we could look to add something in a follow-on release if not in the initial release though? (Since it's a potentially complex feature/set of features)

Reply:
@KristinLBradley yes, I've thought about it, and I've reached your conclusion too: better to consider it in a follow-on release (where we may even have "official" HDS break points at that point)

Reply:
Ok, I think that's a good plan. I think the `Flex` & `Grid` components are still useful even without break points.